### PR TITLE
[full-ci] Enhance create space from ... dialog

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description

image

## Related Issue

- Fixes https://github.com/owncloud/web/issues/8838

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/34950/58/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFilesCopy-copy_feature-L28.png
  


</p></details>
:boom: The oCISFiles5 tests pipeline failed. The build has been cancelled.

File: packages/design-system/src/components/OcModal/OcModal.vue [hunk 1]
```diff
@@ -18,13 +18,18 @@
...
-            <p
+            <div
               v-if="message"
               key="modal-message"
               class="oc-modal-body-message oc-mt-rm"
               :class="{ 'oc-mb-rm': !hasInput }"
-              v-text="message"
-            />
+            >
+              <span v-text="message"></span>
```
Comment (lookacat) [inline packages/design-system/src/components/OcModal/OcModal.vue:27]: `<span v-text="message" />`:)

**Review by lookacat — APPROVED** [review]
One small thing, will approve anyways

**Comment by JanAckermann** [comment]
closed in favor of https://github.com/owncloud/web/pull/8872

**Pull request closed without merging**